app/components/Account/AccountVoting.jsx

```diff
@@ -478,7 +478,7 @@ class AccountVoting extends React.Component {
     _getWorkerArray() {
         let workerArray = [];
 
-        for (let i = 0; i < 100; i++) {
+        for (let i = 0; i >= 0; i++) {
```

Comment:
Having an unbounded for loop is not OK, this will freeze the GUI and trigger thousands of api calls from the ChainStore. I'm gonna implement this properly using the get_all_workers API call, probably gonna add it to the ChainStore itself.

![image](https://user-images.githubusercontent.com/6890015/37762630-27cf00de-2dbd-11e8-8b1b-b09ee27203d0.png)